lib/navigation.ts

```diff
@@ -0,0 +1,53 @@
+export interface NavigationItem {
+  pageName: string;
+  href: string;
+  children?: NavigationItem[];
+}
+
+export const navigation: NavigationItem[] = [
+  { pageName: 'Home', href: '/home', children: [] },
+  {
+    pageName: 'Guide',
+    href: '/guide',
+    children: [
+      { pageName: 'Introduction', href: '/guide/introduction' },
+      {
+        pageName: 'Getting Started',
+        href: '/guide/getting-started',
+      },
+      {
+        pageName: 'Ethereum Provider',
+        href: '/guide/ethereum-provider',
+      },
+    ],
+  },
+];
```

Comment:
We should use the previous way of pulling the routes. It helps with contributors not missing pages in the sidebar/sitemap for the UI, for example this is missing the `mobile` section.

Reply:
Totally with you on this! In this case, I was not done as I wanted to create a full system from pulling routes in a similar way you were doing that would dynamically create this object. 

I was planning to do so on a follow up right after

Reply:
Shane, is this ok to merge, considering that @GuiBibeau will be making those updates on the next PR? I will ensure that it is the next item we get to.